packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -883,6 +883,8 @@ abstract class DiagnosticsNode {
 
         builder.write(
             config.isNameOnOwnLine || description.contains('\n') ? '\n' : ' ');
+        if (description.contains('\n'))
+          builder.prefixOtherLines += '  ';
```

Comment:
how does this look with the dense style?

Reply:
No change, as expected and documented by this test: https://github.com/goderbauer/flutter/blob/7887b12807397ddeb1890dbea30f6e84708e742a/packages/flutter/test/foundation/diagnostics_test.dart#L422